setup.rb

```diff
@@ -108,7 +108,7 @@ module FastlaneRake
   READLINE_VERSION = '6.3'
   READLINE_URL = "http://ftpmirror.gnu.org/readline/readline-#{READLINE_VERSION}.tar.gz"
 
-  RUBY__VERSION = '2.2.4'
+  RUBY__VERSION = '2.4.5'
   RUBY_URL = "http://cache.ruby-lang.org/pub/ruby/2.2/ruby-#{RUBY__VERSION}.tar.gz"
```

Comment:
Ah, I see need for the double-under.

`http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.4.5.tar.gz` returns a 404. `RUBY_URL` looks unused.